Tests/test_Entrez.py

```diff
@@ -142,29 +144,54 @@ def test_construct_cgi_elink3(self):
         self.assertTrue(result_url.startswith(URL_HEAD + "elink.fcgi"), result_url)
         self.assertIn(URL_TOOL, result_url)
         self.assertIn(URL_EMAIL, result_url)
-        self.assertIn("id=15718680", result_url)
-        self.assertIn("id=157427902", result_url)
-        self.assertIn("id=119703751", result_url)
+        self.assertIn("id=15718680%2C157427902%2C119703751", result_url)
```

Comment:
This means the URL is different - does the NCBI treat it the same?

Reply:
This is the test I mentioned above - NCBI does not treat it the same, but I realized I was wrong about how. I'm posting a more detailed comment below.